src/Microsoft.ML.Data/Evaluators/RegressionEvaluator.cs

```diff
@@ -156,6 +158,129 @@ public override void AddColumn(ArrayDataViewBuilder dvBldr, string metricName, p
                 dvBldr.AddColumn(metricName, NumberType.R8, metric);
             }
         }
+
+        public sealed class Result
+        {
+            /// <summary>
+            /// Gets the absolute loss of the model.
+            /// </summary>
+            /// <remarks>
+            /// The absolute loss is defined as
+            /// L1 = (1/m) * sum( abs( yi - y'i))
```

Comment:
>' [](start = 45, length = 1)

 XML escape: &apos;

Reply:
OK. Though the comment does appear to render correctly from [the place where I copied it](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.models.regressionmetrics.l1?view=ml-dotnet#Microsoft_ML_Models_RegressionMetrics_L1).

---
In reply to: [216474189](https://github.com/dotnet/machinelearning/pull/869#discussion_r216474189) [](ancestors = 216474189)